packages/stark-ui/src/modules/app-logout/components/app-logout.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Inject, Input, OnInit, ViewEncapsulation } from "@angular/core";
+
+import {
+	STARK_LOGGING_SERVICE,
+	STARK_ROUTING_SERVICE,
+	STARK_APP_CONFIG,
+	StarkLoggingService,
+	StarkRoutingService,
+	StarkApplicationConfig,
+	STARK_SESSION_SERVICE,
+	StarkSessionService
+} from "@nationalbankbelgium/stark-core";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-app-logout";
+
+/**
+ * Component to display the application's logout button
+ */
+@Component({
+	selector: "stark-app-logout",
+	templateUrl: "./app-logout.component.html",
+	styleUrls: ["./app-logout.component.scss"],
+	encapsulation: ViewEncapsulation.None
+})
+export class StarkAppLogoutComponent implements OnInit {
+	/**
+	 * Desired icon of the logout button
+	 */
+	@Input()
+	public icon?: string;
+
+	/**
+	 * Default icon of the logout button
+	 */
+	public defaultLogoutIcon: string = "power";
+
+	/**
+	 * Class constructor
+	 * @param logger - The logger of the application
+	 * @param routingService - The routing service of the application
+	 * @param sessionService - The session service of the application
+	 * @param appConfig - The configuration of the application
+	 */
+	public constructor(
+		@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService,
+		@Inject(STARK_ROUTING_SERVICE) public routingService: StarkRoutingService,
+		@Inject(STARK_SESSION_SERVICE) public sessionSeervice: StarkSessionService,
```

Comment:
There is a typo in the variable name: "sessionSeervice" => "sessionService"

Reply:
My bad